blog/26-01-2023 - All you need to know about Celo to Ease your Web3 Development Journey/web3-development-made-easy-with-celo

```diff
@@ -0,0 +1,14 @@
+---
+title: All you need to know about Celo to Ease your Web3 Development Journey
+description: In this tutorial, we will introduce you to the Celo blockchain and explain the key features and benefits of using it for your web3 development projects. We will cover the basics of Celo and provide an overview of its technology and ecosystem, as well as discuss the tools and resources available to help you get started. By the end of this tutorial, you will have a better understanding of Celo and how it can support your web3 development journey. Whether you are just starting out or are an experienced developer, this tutorial will provide you with the knowledge you need to get started with Celo.
+authors: 
+  - name: Abdulateef Sanni
+    tittle: Content creation (Video), Celo Sage
+    url: https://www.youtube.com/watch?v=ZpWLdkSTyyc&t=5s
+tags: [celo, celosage, beginner, Web3, Web3 development]
```

Comment:
Use the advised tags, for example, "Celo, Celo Sage, Beginner"

Reply:
I guess the advice tag is issue, when I changed it back to array, the code is cool and no error